### ISSUE TITLE
twisted.spread.sturdy doesn't work at all, not even a little

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2299]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2299|
|Type|defect|
|Created|2006-12-16 01:10:23Z|

Somehow we missed this when eliminating the old service and pb APIs.

Not too surprising - there are zero unit tests.

I guess the module should just be deleted.

<details><summary>Searchable metadata</summary>

```
trac-id__2299 2299
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__pb pb
keywords__ 
time__1166231423000000 1166231423000000
changetime__1166986072000000 1166986072000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

Agreed.

If I recall correctly, even at the time that module was written, it was little more than an experiment.  No applications I'm aware of ever tried to use it.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

Deleted it and the one reference to it (in the glossary) that I could find in remove-sturdy-2299

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)<a name="note_3"></a>|@jerub commented|
|-|-|

Yay for removing pre-UQDS code. Please merge!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [19064]) Merge remove-sturdy-2299

Author: exarkun
Reviewer: jerub
Fixes [#2299](https://github.com/twisted/twisted/issues/2299)

Delete twisted.spread.sturdy, which requires features no longer present
in Twisted to work, if what it did could be called "working" at all.

Also delete the sturdy term from the glossary.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
